cmd/samples/http/receiver/main.go

```diff
@@ -32,14 +33,18 @@ type Example struct {
 	Message  string `json:"message"`
 }
 
-func gotEvent(event cloudevents.Event) {
+func gotEvent(ctx context.Context, event cloudevents.Event, resp *cloudevents.EventResponse) error {
 	fmt.Printf("Got Event Context: %+v\n", event.Context)
 	data := &Example{}
 	if err := event.DataAs(data); err != nil {
 		fmt.Printf("Got Data Error: %s\n", err.Error())
 	}
 	fmt.Printf("Got Data: %+v\n", data)
+
+	fmt.Printf("Got Transport Context: %+v\n", http.TransportContextFrom(ctx))
```

Comment:
look at this @Harwayne ^^

Reply:
That seems to include the HTTP host, path, and method. Nifty.